tests/tableutils.js

```diff
@@ -924,17 +1050,37 @@ describe( 'TableUtils', () => {
 				] ) );
 			} );
 
-			it( 'should properly calculate truncated rowspans', () => {
+			it( 'should move row-spaned cells to a row after removed rows section', () => {
 				setData( model, modelTable( [
-					[ '00', { contents: '01', rowspan: 3 } ],
-					[ '10' ],
-					[ '20' ]
+					[ '00', '01', '02', '03' ],
+					[ { rowspan: 4, contents: '10' }, { rowspan: 3, contents: '11' }, { rowspan: 2, contents: '12' }, '13' ],
+					[ { rowspan: 3, contents: '23' } ],
+					[ '32' ],
+					[ '41', '42' ]
 				] ) );
```

Comment:
This gives strange results in manual test: 
![Zrzut ekranu 2020-04-15 o 13 31 23](https://user-images.githubusercontent.com/1232187/79332645-7d3ab500-7f1d-11ea-9900-f6e0e60ab264.png)
Should be looking like this:
```
+----+----+----+----+
| 00 | 01 | 02 | 03 |
+----+----+----+----+
| 10 | 11 | 12 | 13 |
+    +    +    +----+
|    |    |    | 23 |
+    +    +----+    +
|    |    | 32 |    |
+    +----+----+    +
|    | 41 | 42 |    |
+----+----+----+----+
```
 
![Apr-15-2020 13-35-15](https://user-images.githubusercontent.com/1232187/79332989-12d64480-7f1e-11ea-98ea-ad896af5fb0e.gif)
This seems to be error in some other place (maybe post-fixers?).

Reply:
Welcome to the table world. This is how browsers renders tables :shrug:. I didn't find the talk with some external dev on this but AFAIR we cannot do anything here.

The problem is with "collapsing rows" - just look at the HTML markup - it is OK.

![](https://user-images.githubusercontent.com/247363/79335847-02749880-7f23-11ea-890b-643c3a8d6a45.png)